contracts/HydraChain/modules/Inspector/Inspector.sol

```diff
@@ -137,9 +143,9 @@ abstract contract Inspector is IInspector, ValidatorManager {
 
     /**
```

Comment:
these docs can also be migrated to IInspector.sol

Reply:
Due to a conflict with IInspector interface, which is both in HydraChain and Inspector, this could done by creating an internal function.